desktop/apps/consignments2/client/reducers.js

```diff
@@ -0,0 +1,66 @@
+import * as actions from './actions'
+import ChooseArtist from '../components/choose_artist'
+import CreateAccount from '../components/create_account'
+import DescribeWork from '../components/describe_work'
+import UploadPhoto from '../components/upload_photo'
+import u from 'updeep'
+import { combineReducers } from 'redux'
+import { data as sd } from 'sharify'
+import { last } from 'underscore'
+import { routerReducer } from 'react-router-redux'
+
+const stepsMapping = [
+  {
+    component: CreateAccount,
```

Comment:
Ah, yeah I wasn't sure where it would be cleanest to stick the mapping but I'll give that a try.